.github/workflows/container_base_push.yml

```diff
@@ -79,7 +81,7 @@ jobs:
               uses: docker/setup-qemu-action@v2
             - name: Re-set image tag based on branch
               if: ${{ github.ref_name == 'master' }}
-              run: echo "IMAGE_TAG=stable"
+              run: echo "IMAGE_TAG=stable" >> $GITHUB_ENV
```

Comment:
As discussed, can we please change this from "stable" to something else? In the issue below I'm suggesting "alpha":

- #9539